### ISSUE TITLE
Create a gif with the last 5 or 10 pictures

### ISSUE THREAD

**Issue by dns2utf8** [coredump-ch/coredumpbot#42]
The bot would send a serie of pictures or a rendered gif.

**Comment by dbrgn** [comment]
I like that idea, but it means that it constantly needs to stream the webcam data stream and assemble GIFs, right?

**Comment by dns2utf8** [comment]
No, it could just start collecting on request. The user would have to wait a minute or two.

**Comment by dbrgn** [comment]
Yeah, why not! The bot could send a message when starting to process the request, and one when it's finished.
